components/Dropdown/Dropdown.js

```diff
@@ -2,24 +2,23 @@ class Dropdown {
   constructor(element) {
     
     // Assign this.element to the dropdown element
-    this.element;
+    this.element = element.querySelector('.dropdown');
     
     // Get the element with the ".dropdown-button" class found in the dropdown element (look at the HTML for context)
-    this.button = this.element.querySelector();
+    this.button = element.querySelector('.dropdown-button');
     
     // assign the reference to the ".dropdown-content" class found in the dropdown element
-    this.content;
+    this.content = element.querySelector('.dropdown-content');
     
     // Add a click handler to the button reference and call the toggleContent method.
-    this.button.addEventListener('click', () => {
-
-    })
+    this.button.addEventListener('click', () =>this.toggleContent());
   }
 
   toggleContent() {
     
     // Toggle the ".dropdown-hidden" class off and on
-    this.content;
+    this.content.classList.toggle('dropdown-open');
+    this.content.classList.toggle('dropdown-hidden');
   }
 }
 
```

Comment:
you also should create a dropdown variable and grab the dropdown with querySelectorAll and run the forEach on it after